spec/lib/whitehall_importer/migrate_assets_spec.rb

```diff
@@ -109,7 +128,10 @@
     end
 
     it "should delete live image variants that have no content publisher equivalent" do
-      asset = build(:whitehall_migration_asset_import, :for_image, variant: "s216")
+      asset = build(:whitehall_migration_asset_import,
+                    :for_image,
+                    variant: "s216",
+                    original_asset_url: "https://static.gov.uk#{asset_path}")
```

Comment:
I guess this is being set on each item because of the http request on each model? I assume this can be avoided if we move the HTTP request

Reply:
I've reworked the tests in light of the changes to the code, and as such setting these URLs is redundant.